package Array;

public class Test4 
{

	public static void main(String[] args)
	{	
		//Step-1: Array Declaration
			char[] ar2=new char[5];
		
		//Step-2: Array Initilisation/Assignment
			ar2[0]='A';
			ar2[1]='b';
			ar2[2]='x';
			ar2[3]='y';
			ar2[4]='Z';
			ar2[5]='r';
		
		//Step-3: Usage
		System.out.println(ar2[0]);//A	
		System.out.println(ar2[1]); //b
		System.out.println(ar2[2]);// x
		System.out.println(ar2[3]);// y
		System.out.println(ar2[4]);  //Z
		//System.out.println(ar2[5]);  // java.lang.ArrayIndexOutOfBoundsException
	
	}
	
}
